metakb/transform/civic.py

```diff
@@ -651,7 +651,7 @@ def _get_disease_descriptors(self, disease):
             if disease_norm_resp['match_type'] > highest_match:
                 highest_match = disease_norm_resp['match_type']
                 normalized_disease_id = \
-                    disease_norm_resp['value_object_descriptor']['value']['disease_id']  # noqa: E501
+                    disease_norm_resp['value_object_descriptor']['value']['id']  # noqa: E501
```

Comment:
You should update the requirements with the updated version for disease-normalizer.